envs/prod/templates/team-geordi.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: team-geordi
+  namespace: argocd
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: yelb
+    server: {{ .Values.spec.destination.server }}
+  source:
+    repoURL: https://github.com/mreferre/yelb.git
+    path: deployments/platformdeployment/Kubernetes/chart/yelb
```

Comment:
must point to teams/team-geordi/prod, not yelb

Reply:
Having Dev to review, before i do test, prod. This is done now.